File: python/ex008.py
# Desenvolva um programa que leia uma distância em metros e mostre os valores
# relativos em outras medidas.
# Ex:
# Digite uma distância em metros: 185.72
# A distância de 85.7m corresponde a:
# 0.18572Km
# 1.8572Hm
# 18.572Dam
# 1857.2dm
# 18572.0cm
# 185720.0mm

number = float(input("Digite uma distância em metros: "))

kilometro = number/1000
hectometro = number/100
decametro = number/10
decimetro = number*10
centimetro = number*100
milimetro = number*1000

print(f"A distância de {number} corresponde a: \n {kilometro}Km \n {hectometro}Hm \n {decametro}Dam \n {decimetro}dm \n {centimetro}cm \n {milimetro}mm")

File: typescript/src/ex008.ts
// 8) Desenvolva um programa que leia uma distância em metros e mostre os valores
// relativos em outras medidas.
// Ex:
// Digite uma distância em metros: 185.72
// A distância de 85.7m corresponde a:
// 0.18572Km
// 1.8572Hm
// 18.572Dam
// 1857.2dm
// 18572.0cm
// 185720.0mm

var readline = require('readline-sync')

let distancMetros : number = Number(readline.question("Digite uma distancia em metros: "));

let kilometro : number = distancMetros / 1000;
let hectometro : number = distancMetros / 100;
let decatometro : number = distancMetros / 10;
let decimetro : number = distancMetros * 1000;
let centimetro : number = distancMetros * 100;
let milimetro : number = distancMetros * 10;

console.log(`A distância de ${distancMetros} corresponde a: `)
console.log(`${kilometro}Km \n ${hectometro}Hm \n ${decatometro}Dam \n ${decimetro}dm \n ${centimetro}cm \n ${milimetro}mm`);

